### ISSUE TITLE
Proximity world

### ISSUE THREAD

**Issue by levitskyyy** [Wiicart/Strings#11]
how to use the world the player is in?

if I delete `world: "world" `- the function does not work

**Comment by Wiicart** [comment]
Right now a proximity channel needs a world defined.  I plan to change this in the future, but in the meantime, if you want a proximity channel it only works in one world.  If you need a proximity channel for multiple worlds, you'll have to make one for each world.

**Comment by levitskyyy** [comment]
Please add support for the world the player is currently in

**Comment by Wiicart** [comment]
That is planned, but like the other things, I won't be able to publish an update until after Nov. 1

**Comment by AZsSPC** [comment]
the simpliest fix i see i making `world: "world"` an array, so tech.asmins can choose all worlds for chat.
also a small feature - add placeholders for chats for each world like that:
`proxiworld:
    ***
    worlds: |
        world: '[world prefix]'
        second_world: '[second world prefix]'
        world_the_end: '[end prefix]'`
![image](https://github.com/user-attachments/assets/27f329e5-0c5b-4de9-9995-01d056f54333)
as i know thats not hart to implement

**Comment by Wiicart** [comment]
Yeah I'll do something like that, though for the prefixes I'm not sure.  I kind of want to limit each channel to one prefix as that kind of ventures in to sub-channels which I'm not really sure I want, just for simplicity.

**Comment by AZsSPC** [comment]
there also ways to fork and do own features for anyone who want and can, so not a problem :)

**Comment by Wiicart** [comment]
👍, there’s also the API. https://github.com/Wiicart/Strings/tree/main/strings-api

I think I’m going to add a way for custom channel implementations soon, so that’ll also be an option.

**Comment by Wiicart** [comment]
Multi-world channels have been implemented
